packages/matter.js/tools/util/TsFile.ts

```diff
@@ -0,0 +1,453 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Specification } from "../../src/model/index.js";
+import { writeMatterFile } from "./file.js";
+import { asObjectKey, wordWrap } from "./string.js";
+
+const HEADER = `/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*** THIS FILE IS GENERATED, DO NOT EDIT ***/`;
+
+const INDENT = "    ";
+const WRAP_WIDTH = 80;
```

Comment:
if you like increase ... 120 are fully ok in my eyes these days

Reply:
Yeah OK, that's what I'm generally wrapping the code at anyway

Reply:
hehe me too